dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/PluginFeaturesService.kt

```diff
@@ -112,6 +118,39 @@ internal abstract class PluginFeaturesService : BuildService<PluginFeaturesServi
         }
     }
 
+    internal val enableK2Analysis: Boolean by lazy(SYNCHRONIZED) {
```

Comment:
Nit: default for `lazy` is `SYNCHRONIZED`: https://kotlinlang.org/docs/delegated-properties.html#lazy-properties

Reply:
TIL. Do you think it's better to keep `SYNCHRONIZED` it to be explicit, or remove it?